onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -78,7 +77,10 @@ TensorrtExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
   int counter = 0;
   for (const auto& group : supported_nodes_vector) {
     if (!group.empty()) {
-      std::set<size_t> node_set(group.begin(), group.end());
+      std::set<size_t> node_set;
```

Comment:
you can reserve the size of this set in advance using node_set.reserve(supported_nodes_vector.size())

Reply:
set doesn't have reserve method. But I've reserved the size for unordered_map etc in the code.